mmengine/optim/optimizer/amp_optimizer_wrapper.py

```diff
@@ -151,6 +151,12 @@ def load_state_dict(self, state_dict: dict):
         """
         if 'loss_scaler' in state_dict:
             self.loss_scaler.load_state_dict(state_dict.pop('loss_scaler'))
+
+        if 'base_param_settings' in state_dict:
+            self.base_param_settings = state_dict.pop('base_param_settings',
+                                                      None)
```

Comment:
```suggestion
            self.base_param_settings = state_dict.pop('base_param_settings')
```

Reply:
Suggest following the logic in https://github.com/open-mmlab/mmengine/pull/1019/files#diff-8ada679358329e82655730d1897deece9576e5f9844b3086da257c8afdb27a24R289

Reply:
Thanks for your suggestion, there should not return none after conditional check, therefore remove fallback pop none.